A sesam devcontainer based on MS python and docker-in-docker examples

mofsesam opened the pull request:
A profe of consept but fully usable .devcontainer for sesam development with Visual studio code - remote extention and docker.

This is one way of setting up a consistant development invironment and support workflow across different OS, mac/windows/linux.

mofsesam committed:
A sesam devcontainer based on MS python and docker-in-docker examples

BaardBouvet reviewed (changes requested):
Remember to add an OSS license (Apache-2.0 preferably)

mofsesam committed:
Create LICENSE

In line with https://github.com/microsoft/vscode-dev-containers/blob/master/LICENSE

mofsesam committed:
Merge pull request #1 from mofsesam/add-license-1

Create LICENSE

mofsesam committed:
Update README.md

mofsesam commented:
@BaardBouvet since it is based on examples with license:
https://github.com/microsoft/vscode-dev-containers/blob/master/LICENSE
I added a MIT license. I think that is the simplest.
